### ISSUE TITLE
Question on dataloader

### ISSUE THREAD

**Issue by AnqiaoLi** [jayLEE0301/vq_bet_official#3]
Hi! Thank you for the excellent work! I have two questions on the data loader. 
- First, when I read the code in the class `TrajectorySlicerDataset`, I found [the code](https://github.com/jayLEE0301/vq_bet_official/blob/09d4851288ca5deaaa1ab367a208e520f8ee9a84/examples/dataset.py#L461C1-L470C18) under option `self.vqbet_get_future_action_chunk == False` is the same in both `if` and `else`. Is this a typo or some kind of design?
- Second, is the `self.vqbet_get_future_action_chunk` set to `False` if and only if when pretraining the VQ-VAE? If so, should [this line](https://github.com/jayLEE0301/vq_bet_official/blob/09d4851288ca5deaaa1ab367a208e520f8ee9a84/examples/dataset.py#L370C1-L371C1) be changed to `min_window_required = action_window` instead of `min_window_required = max(window, action_window)` to ensure every slice of length equals `action_window` is fully used to train the VQ-VAE?

**Comment by jayLEE0301** [comment]
As you said, if / else in that line can be combined into one.

Regarding the second question, technically the action chunks at the end of the episode aren't critical to the success of the task, so it shouldn't cause a problem, but if you're using a demo that has meaningful action until the end of the episode, I'd recommend modifying it.

We'll update the code with a fix soon, thank you.

**Comment by AnqiaoLi** [comment]
Thanks for the reply. And I have another question. In class [VectorQuantize](https://github.com/jayLEE0301/vq_bet_official/blob/09d4851288ca5deaaa1ab367a208e520f8ee9a84/vector_quantize_pytorch/vector_quantize_pytorch.py#L706C1-L707C1), the `orthogonal_reg_weight` is set to 0 for all experiments as default. However, in the paper, the orthogonal loss is added, which I think should be the second term of the $\mathcal{L}_{RVQ}$. 
Is there a mismatch?
![image](https://github.com/jayLEE0301/vq_bet_official/assets/78886956/4adb1c9d-1265-4b2d-8f30-aa17137898ed)

**Comment by jayLEE0301** [comment]
The second term in the Eq L_RVQ is not updated via orthogonal loss, but via "ema_update". Please check 
https://github.com/jayLEE0301/vq_bet_official/blob/main/vector_quantize_pytorch/vector_quantize_pytorch.py#L657

**Comment by AnqiaoLi** [comment]
Thanks for your explanation!
